### DISCUSSION TITLE
Request to Cancel GitHubCopilot Trial Subscription

### DISCUSSION THREAD

**Discussion by DangDuyKhang1999** [community/community#101134] [Copilot Conversations]
Dear GitHub Support Team,

I hope this email finds you well. My name is Khang, and I am writing to request the cancellation of the GitHub Copilot trial subscription associated with my email address.

I recently received a 30-day trial invitation for GitHub Copilot at my company email address. Unfortunately, due to some constraints, I am unable to use my company account to log in to GitHub and activate the trial. This is primarily because of certain policies and restrictions within my company that prevent the use of company accounts for such purposes.

I understand the value and potential of GitHub Copilot, but regretfully, I won't be able to explore its features during the trial period. Therefore, I kindly request your assistance in canceling the trial subscription linked to my email address ([KhangDD4@fpt.com](mailto:KhangDD4@fpt.com)).

I apologize for any inconvenience this may cause, and I appreciate your understanding in this matter. If there are any further steps or information required from my side to process this request, please do not hesitate to let me know.

Thank you for your prompt attention to this matter, and I appreciate your assistance in resolving it.

Best regards,

Dang Duy Khang

**Comment by mishmanners** [comment]
Thanks for reaching out. You can try disabling GitHub Copilot from your settings:
1. Go to GitHub Profile
2. Click your profile picture in the top right
3. From the drop down menu select "Settings"
4. On the left pane menu, scroll down and click "Copilot" under "Code, planning, automation"

You should be able to disable or change your GitHub Copilot status from there.

Let me know if that works.

**Comment by DangDuyKhang1999** [comment]
Hello Mishmanners

I hope you are having a good day.

I am currently experiencing some issues and would appreciate your assistance.

- Presently, I am unable to log in to GitHub with the account khangdd4@fpt.com due to company login policies. I would like guidance on resolving this issue.

- I believe I have never used the email address khangdd4@fpt.com to create a GitHub account, yet I received an email notifying me of a successful registration for GitHub Copilot. Is it possible that Visual Studio automatically registered GitHub Copilot when I installed this extension?

- After the GitHub Copilot trial period expires, does the service automatically transition to a personal plan? I would like to know more about this process.

I look forward to your assistance and clarification. Thank you!

Best regards.
